manifests/backup/mysqlbackup.pp

```diff
@@ -0,0 +1,85 @@
+# See README.me for usage.
+define mysql::backup::mysqlbackup (
```

Comment:
No, I don't think so. The full name is MySQL Enterprise Backup and the
package name is meb and the tool/binary which you actually need to call
is mysqlbackup.

On 02/05/2015 07:36 PM, Hunter Haugen wrote:
>
> In manifests/backup/mysqlbackup.pp
> <https://github.com/puppetlabs/puppetlabs-mysql/pull/649#discussion_r24186509>:
>
> > @@ -0,0 +1,85 @@
> > +# See README.me for usage.
> > +define mysql::backup::mysqlbackup (
>
> Should this be called mysqlenterprisebackup instead?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/puppetlabs/puppetlabs-mysql/pull/649/files#r24186509>.
>